docs/Hyprland.1.rst

```diff
@@ -0,0 +1,75 @@
+========
+Hyprland
+========
+
+---------------------------------
+Dynamic tiling Wayland compositor
+---------------------------------
+
+:Date: 15 Jul 2022
+:Copyright: Copyright (c) 2022, vaxerski
+:Version: 0.7.1beta
```

Comment:
This is yet another location where version is used. Can we set it through something like an import? maybe we could keep it by default `@VERSION@`, and only change it using a script on release.

All the places in this repo where we use the version string could benefit from having a single place to edit it.

Reply:
I like all your suggestions. As for `@Version@` I'm okay with doing that, but not sure of how you want to go about implementing that. I do agree though where ever possible version strings should be interpolated

Reply:
That was more of a long-term idea. For this PR you can keep the current version, and we'll figure out the versioning system in another one.

Reply:
Okay so I think this one can be resolved. Having the ability to input the version in the make file will make things way easier later.